experimental/BasisLieHighestWeight/src/ModuleData.jl

```diff
@@ -0,0 +1,80 @@
+abstract type ModuleData end
+
+# To be implemented by subtypes:
+# Mandatory:
+#   base_lie_algebra(V::MyModuleData) -> LieAlgebra
+#   highest_weight(V::MyModuleData) -> WeightLatticeElem
+#   dim(V::MyModuleData) -> ZZRingElem
+#   character(V::MyModuleData) -> Dict{WeightLatticeElem,ZZRingElem}
+
+
+mutable struct SimpleModuleData <: ModuleData
+    base_lie_algebra::LieAlgebra
```

Comment:
```suggestion
    L::LieAlgebra
```
the field name can just stay as `L`. My comment was just about the name of the getter functions